README.md

```diff
@@ -4,13 +4,37 @@
 
 > This project is meant to be an HTML & CSS exercise from the Microverse Program's Module 1.
 
+![screenshot](desktop_screenshot.png)
+> Desktop version
+
+![screenshot](mobile_screenshot.png)
+> Mobile version
+
+The project consists in a mobile and desktop skeleton for a portfolio webpage. Feel free to try it out and explore the code as much as you like!
 
-The project consists in a mobile version skeleton of a portfolio webpage.
 
 ## Built With
 
 - HTML5 & CSS3
 
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] It would be great if the deploy link appears on the readme file, I kindly invite you to add it. 🐶 
- After this, I'll be able to check the deployment.

Reply:
Change request completed!